cmd/fdroidcl/search.go

```diff
@@ -190,6 +195,23 @@ func filterAppsUpdates(apps []fdroidcl.App, device *adb.Device) []fdroidcl.App {
 	return result
 }
 
+func filterAppsAge(apps []fdroidcl.App, days int) []fdroidcl.App {
+	var result []fdroidcl.App
+	older := false
```

Comment:
I'd prefer that you flipped this and used `==` instead of `!=` below, to simplify the logic a little.